### DISCUSSION TITLE
Copy/Import/Export NPC's

### DISCUSSION THREAD

**Discussion by KMUR9** [fmasa/wfrp-master#357] [Ideas]
Just wondered if there is any way to copy NPC's and/or PC's between different parties without having to rebuild them completely. Or some form of import/export to a .json file like with compendium skills etc. 
If not this would be a really useful feature to have in the future.
Cheers

**Comment by fmasa** [comment]
@KMUR9 I would love to add this, I was lately duplicating some NPCs myself between parties and it was painful!

I think supporting the JSON is probably the easiest and the most versatile approach. 👍

**Comment by fmasa** [comment]
Tracking now in https://github.com/fmasa/wfrp-master/issues/388
